src/serial.zig

```diff
@@ -970,69 +961,26 @@ fn mapBaudToMacOSEnum(baudrate: usize) !std.os.darwin.speed_t {
     };
 }
 
-const DCBFlags = struct {
-    fBinary: u1, // u1
-    fParity: u1, // u1
-    fOutxCtsFlow: u1, // u1
-    fOutxDsrFlow: u1, // u1
-    fDtrControl: u2, // u2
-    fDsrSensitivity: u1, // u1
-    fTXContinueOnXoff: u1, // u1
-    fOutX: u1, // u1
-    fInX: u1, // u1
-    fErrorChar: u1, // u1
-    fNull: u1, // u1
-    fRtsControl: u2, // u2
-    fAbortOnError: u1, // u1
+const DCBFlags = packed struct(u32) {
+    fBinary: u1 = 1, // u1
+    fParity: u1 = 0, // u1
+    fOutxCtsFlow: u1 = 0, // u1
+    fOutxDsrFlow: u1 = 0, // u1
+    fDtrControl: u2 = 1, // u2
+    fDsrSensitivity: u1 = 0, // u1
+    fTXContinueOnXoff: u1 = 0, // u1
+    fOutX: u1 = 0, // u1
+    fInX: u1 = 0, // u1
+    fErrorChar: u1 = 0, // u1
+    fNull: u1 = 0, // u1
+    fRtsControl: u2 = 0, // u2
+    fAbortOnError: u1 = 0, // u1
```

Comment:
you can use use `bool` here which removes the need for the `if`s above :)

Reply:
I actually tried removing the `@as` before because i didnt like it either... 
Didn't know that `bool` will be equvalent to `u1`, im testing it now and pushing a commit if it works :)